bot/seasons/christmas/hanukkah_embed.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+import logging
+
+import requests
+from discord import Embed
+from discord.ext import commands
+
+
+log = logging.getLogger(__name__)
+
+
+class HanukkahEmbed(commands.Cog):
+    """
+    A cog that returns information about Hanukkah festival.
+    """
+    def __init__(self, bot):
+        self.bot = bot
+        self.url = """https://www.hebcal.com/hebcal/?v=1&cfg=json&maj=on&min=on&mod=on&nx=on&year=now&month=x&ss=on&
```

Comment:
Use parenthesis here instead of triple quotes to be more consistent with the rest of the code.

Reply:
okay